packages/table/package.json

```diff
@@ -46,7 +46,8 @@
     "@leafygreen-ui/button": "workspace:^",
     "@leafygreen-ui/badge": "workspace:^",
     "@leafygreen-ui/pagination": "workspace:^",
-    "@leafygreen-ui/tooltip": "workspace:^"
+    "@leafygreen-ui/tooltip": "workspace:^",
+    "@emotion/styled": "^11.14.0"
```

Comment:
What are we using styled for?

Reply:
I'm using it in the spec file. I was getting an error locally about it not being included. However, I noticed that polymorphic also uses styled in the spec file, but it's not included in package.json. Maybe I don't need it.